### ISSUE TITLE
Stratford: Accessibility - HTML Validation Error

### ISSUE THREAD

**Issue by ivan-ottinger** [Automattic/themes#1473]
The theme doesn't seem to be fully WCAG AA compliant - even it is tagged as accessibility-ready.

It is not clear to me how the "accessibility-ready" term is defined, but I think it would be worth if we could fix the HTML validation error which is mentioned below.

## Steps to replicate

Activate the Stratford theme and run validation test like [https://validator.w3.org](https://validator.w3.org).

What we will see in the report is that the `main` element has a `section` as a parent element which shouldn't be the case. 

## Result

Theme is not WCAG AA compliant / there's a HTML validation error.

## Expected

Theme is WCAG AA compliant  / there's no HTML validation error.

* Follow-up ticket: #2379839-zen
* thread / P2: p4vt7e-5O-p2

**Comment by danieldudzic** [comment]
Issues highlighted by the validator are not caused by the theme.

It seems that a Reblog box causes the validation to fail: https://en.support.wordpress.com/reblogs/
